.env

```diff
@@ -0,0 +1,8 @@
+VITE_PUBLIC_FIREBASE_API_KEY="AIzaSyCa0AttF5oy6I0pXpRFvwmlf8fUOnLINtM"
```

Comment:
Put in gitignore. Add .env.sample for ref

Reply:
done. https://github.com/dscnitrourkela/project-vanilla/pull/53/commits/9475c1f3b08d461c99e6f38ea4cca00d0ac2bbcd